portal-frontend/src/app/features/notice-of-intents/notice-of-intent-details/pfrs-details/pfrs-details.component.html

```diff
@@ -35,39 +35,28 @@
 
   <div class="subheading2 grid-1">Placement of Fill Project Duration</div>
   <div class="grid-double">
-    <div *ngIf="_noiSubmission.fillProjectDurationAmount && _noiSubmission.fillProjectDurationUnit">
-      {{ _noiSubmission.fillProjectDurationAmount }}
-      <ng-container [ngPlural]="_noiSubmission.fillProjectDurationAmount">
-        <ng-template ngPluralCase="=1">
-          {{ _noiSubmission.fillProjectDurationUnit.slice(0, -1) | titlecase }}
-        </ng-template>
-        <ng-template ngPluralCase="other">
-          {{ _noiSubmission.fillProjectDurationUnit | titlecase }}
-        </ng-template>
-      </ng-container>
+    <div *ngIf="_noiSubmission.fillProjectDuration">
+      {{ _noiSubmission.fillProjectDuration }}
     </div>
     <app-no-data
       [showRequired]="showErrors"
-      *ngIf="!_noiSubmission.fillProjectDurationAmount || !_noiSubmission.fillProjectDurationUnit"
+      *ngIf="!_noiSubmission.fillProjectDuration"
     ></app-no-data>
   </div>
 
   <div class="subheading2 grid-1">Removal of Soil Project Duration</div>
   <div class="grid-double">
-    <div *ngIf="_noiSubmission.soilProjectDurationAmount && _noiSubmission.soilProjectDurationUnit">
-      {{ _noiSubmission.soilProjectDurationAmount }}
-      <ng-container [ngPlural]="_noiSubmission.soilProjectDurationAmount">
-        <ng-template ngPluralCase="=1">
-          {{ _noiSubmission.soilProjectDurationUnit.slice(0, -1) | titlecase }}
-        </ng-template>
-        <ng-template ngPluralCase="other">
-          {{ _noiSubmission.soilProjectDurationUnit | titlecase }}
+    <div *ngIf="_noiSubmission.soilProjectDuration">
+      {{ _noiSubmission.soilProjectDuration }}
+      <ng-container>
+        <ng-template>
+          {{ _noiSubmission.soilProjectDuration | titlecase }}
         </ng-template>
       </ng-container>
```

Comment:
Does not need ng-container, ng-template, or titlecase pipe anymore

Reply:
removed